docs/website/docs/getting-started/tflite/python.md

```diff
@@ -0,0 +1,78 @@
+# TFLite via Python
+
+The example below demonstrates downloading, compiling, and executing a TFLite
+model using the Python API. This includes some initial setup to declare global
+variables, download the sample module, and download the sample inputs.
+
+Declaration of absolute paths for the sample repo and import all required libraries.
+The default setup uses the CPU backend as the only target. This can be reconfigured
+to select alternative targets.
+
+```
+import iree.compiler.tflite as iree_tflite_compile
+import iree.runtime as iree_rt
+import numpy
+import os
+import urllib.request
+
+from PIL import Image
+
+workdir = "/tmp/workdir"
+os.makedirs(workdir, exist_ok=True)
+
+tflite = "/".join([workdir, "model.tflite"])
+jpg = "/".join([workdir, "input.jpg"])
```

Comment:
These names are pretty short and could overlap with package names. Change to "tfliteFile" and "jpgFile"?

Reply:
Done.